feat: Use shinylive assets v0.5.0 (#108)
posit-dev/r-shinylive: NEWS.md

# shinylive (development version)

> * shinylive now uses [shinylive web assets v0.5.0](https://github.com/posit-dev/shinylive/releases/tag/v0.5.0) by default. This update brings improved keyboard shortcuts for R users in the Shinylive editor, the ability to export a static, custom library of R packages with the exported app, and a few improvements to the Quarto integration, along with an upgrade to webR 0.4.0 with R 4.4.1. (#108)

Author: ```suggestion
* shinylive now uses [shinylive web assets v0.5.0](https://github.com/posit-dev/shinylive/releases/tag/v0.5.0) by default, which bundles webR 0.4.0 with R 4.4.1. This update brings improved keyboard shortcuts for R users in the Shinylive editor, the ability to export a custom library of R packages with the exported app, and a few improvements to the Quarto integration. (#108)
```
